cpp/ps0007/150daysprogramming.cpp

```diff
@@ -26,158 +10,64 @@ float area(char shape, float dimensions[]);
 
 int main()
 {
-
-    // prompting user for shape and dimensions 
+    // prompting user for shape and dimensions 1.5/1.5
     char ch;
-    cout << "enter the shape char : \n" << "c for circle: \n r for rectangle: \n s for square: \n t for triangle: \n ";
+    cout << "enter the shape char : \n c for circle: \n r for rectangle: \n s for square: \n t for triangle: \n ";
     cin >> ch;

```

Comment:
- [x] declared variables to store the user input
- [x] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input